### DISCUSSION TITLE
Render Notebook inside of another notebook

### DISCUSSION THREAD

**Discussion by jeremiahpslewis** [fonsp/Pluto.jl#1915] [Questions about using Pluto]
If I have a series of mini-notebooks (‘PlutoBites’), can I ‘import’ them into another notebook to be rendered there? E.g. if I have answers to a problem set, where each question uses similar variables, I might use separate notebooks to not get a ‘multiple definitions’ error, but then want to export / present the result as a single set of answers. Or likewise if I have a module like ‘how to install Julia’, I might want to reuse it in a number of different notebooks without copy-paste.

**Comment by greimel** [comment]
This would be extremely helpful for teaching materials in lectures that build up over time.

Some wishes that come to my mind.

* it would become visually clear that the notebook is `include`d so that reader can skip it if already familiar (similar to the idea of a `module`). Example: include functionality developed in the previous lecture.
* The included notebook should be embedded in the export html, so that notebooks remain self-contained.
